tests/NuGet.IndexingTests/TestSupport/MockTopDocs.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Lucene.Net.Search;
+
+namespace NuGet.IndexingTests.TestSupport
+{
+    public class MockTopDocs : TopDocs
+    {
+        public MockTopDocs(int totalHits, ScoreDoc[] scoreDocs, float maxScore) :
+            base(totalHits, scoreDocs, maxScore)
```

Comment:
After thinking about it, this sufficiently works around having to add a new constructor, and allows me to make some setup for testing purposes as well, which I am actually against adding in non-test code. I will leave this as is.